src/addons/addonapi.cpp

```diff
@@ -94,7 +94,7 @@ QJSValue AddonApi::settings() const {
 QJSValue AddonApi::navigator() const {
   QJSEngine* engine = QmlEngineHolder::instance()->engine();
 
-  QObject* obj = SettingsHolder::instance();
+  QObject* obj = Navigator::instance();
```

Comment:
Would it be possible to also add test cases that check this?

Reply:
because the test is silly: it checks if `vpn.navigator` exists but it does not call method because the navigator is not available in unit-tests.

Reply:
:/ Alright!